docs/source/02_get_started/03_usage.md

```diff
@@ -17,38 +17,52 @@ On your terminal run:
 
 ```shell
 mkdir qhub-test && cd qhub-test
-``` 
+```
 
 #### Fully automated deployment
 
 To generate a fully automated deployment, on your terminal run:
 
 ```shell
-qhub init aws --project project-name --domain jupyter.qhub.dev --ci-provider github-actions --auth-provider auth0 
+qhub init aws --project projectname --domain qhub.dev --ci-provider github-actions --auth-provider auth0
 --auth-auto-provision --repository github.com/quansight/project-name --repository-auto-provision
 ```
 
 The command above will generate the `qhub-config.yaml` config file
-with an infrastructure deployed on `aws`, named `project-name`, where
-the domain will be `jupyter.github-actions.qhub.dev`. The deployment
+with an infrastructure deployed on `aws`, named `projectname`, where
+the domain will be `qhub.dev`. The deployment
 will use `github-actions` as the continuous integration (CI) provider,
 automatically provisioned and authenticated by `auth0`, initialized on
-GitHub under the URL `github.com/quansight/project-name `.
+GitHub under the URL `github.com/quansight/projectname`.
 
 There are several **optional** (yet highly recommended) flags that
 allow to configure the deployment:
 
-+ `aws` indicates that the project will be deployed on the Amazon AWS Cloud provider.
-- `--project`: project-name is required to be a string compliant with the Cloud provider recommendations (see official Cloud provider docs on naming policies).
+- `aws` indicates that the project will be deployed on the Amazon AWS Cloud provider.
+    + Optional flags are: `gcp`, `do` and `azure`.
+- `--project`: the name of the project is required to be a string compliant with the Cloud provider recommendations. For
+  more details see official Cloud provider docs on naming policies and check our docs on [naming convention](#project-naming-convention).
+- `--domain`: base domain for your cluster. This pattern is also applicable if you are setting your own DNS through a different provider.
   + `jupyter.qhub.dev` is the domain registered on CloudFlare. In case you chose not to use Cloudflare, skip this flag.
```

Comment:
```suggestion
  + `qhub.dev` is the domain registered on CloudFlare. In case you chose not to use Cloudflare, skip this flag.
```
Have this changed too?

Reply:
I did indeed :) Thanks again for flagging it!